src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.diff.sql

```diff
@@ -169,6 +169,7 @@ CREATE TYPE dbo.AddCustomTagsInputTableType_1 AS TABLE
 (
     TagPath                    VARCHAR(64),  -- Custom Tag Path. Each custom tag take 8 bytes, support upto 8 levels, no delimeter between each level.
     TagVR                      VARCHAR(2),  -- Custom Tag VR.
+    TagPrivateCreator          NVARCHAR(64),  -- Custom Tag Private Creator, only valid for private tag.
```

Comment:
you should check the length during validation or do all the checks done by VR.LO. Because Private data element is of type LO. Check for control character and all